src/inc/til/bitmap.h

```diff
@@ -119,7 +113,8 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
                 else
                 {
                     // If we reached the end, set the pos because the run is empty.
-                    _pos = _nextPos;
+                    _pos = _end;
+                    _nextPos = _end;

```

Comment:
`find_next` returns `2^63-1` if it was unable to find the next-set-bit. Due to that `_nextPos > _end` is possible.

Reply:
This seems like a valuable comment to leave in the code IMO

Reply:
Yeah, leave that comment here and it's good for me.